clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListMetalakes.java

```diff
@@ -50,13 +51,12 @@ public void handle() {
       return;
     }
 
-    List<String> metalakeNames = new ArrayList<>();
+    List<String> headers = Arrays.asList("metalake");
+    List<List<String>> rows = new ArrayList<>();
     for (int i = 0; i < metalakes.length; i++) {
-      metalakeNames.add(metalakes[i].name());
+      rows.add(Arrays.asList(metalakes[i].name()));
     }
-
-    String all = Joiner.on(System.lineSeparator()).join(metalakeNames);
-
-    System.out.println(all.toString());
+    TablePrinter tablePrinter = new TablePrinter();
+    tablePrinter.print(headers, rows);
```

Comment:
This needs to be under a flag and not change an existing commands output.

Reply:
hi @justinmclean 
I think we only need one format output, why do we needs kept two different output?
Can you explain?